chains/v20/preConfigured/detailsDev/6a2f9e22feb677ca03b1887493ad5eeb240d6133b5b89999b76ab3695a036c64.json

```diff
@@ -1,19 +0,0 @@
-{
-    "chainId": "6a2f9e22feb677ca03b1887493ad5eeb240d6133b5b89999b76ab3695a036c64",
```

Comment:
would it work properly, if chains.json won't update yet, but this file will be removed (request to chain will fail with an error)?